server/datastore/mysql/vulnerabilities.go

```diff
@@ -16,44 +16,59 @@ func (ds *Datastore) Vulnerability(ctx context.Context, cve string, teamID *uint
 	var vuln fleet.VulnerabilityWithMetadata
 
 	eeSelectStmt := `
-		SELECT
-			vhc.cve,
-			MIN(COALESCE(osv.created_at, sc.created_at, NOW())) AS created_at,
+		SELECT DISTINCT
+			cm.cve,
+			COALESCE(LEAST(osv.created_at, sc.created_at), NOW()) AS created_at,
 			COALESCE(osv.source, sc.source, 0) AS source,
 			cm.cvss_score,
 			cm.epss_probability,
 			cm.cisa_known_exploit,
 			cm.published,
-			COALESCE(cm.description, '') AS description,
-			vhc.host_count,
-			vhc.updated_at as host_count_updated_at
-		FROM
-			vulnerability_host_counts vhc
-		LEFT JOIN cve_meta cm ON cm.cve = vhc.cve
-		LEFT JOIN operating_system_vulnerabilities osv ON osv.cve = vhc.cve
-		LEFT JOIN software_cve sc ON sc.cve = vhc.cve
-		WHERE vhc.cve = ?
-	`
-	eeGroupBy := " GROUP BY vhc.cve, source, cm.cvss_score, cm.epss_probability, cm.cisa_known_exploit, cm.published, description, vhc.host_count, host_count_updated_at"
+			cm.description,
+			COALESCE(vhc.host_count, 0) as host_count,
+			COALESCE(vhc.updated_at, NOW()) as host_count_updated_at
+		FROM cve_meta cm
+		JOIN (
+			SELECT cve
+			FROM software_cve
+			WHERE cve = ?
+			
+			UNION
+			
+			SELECT cve
+			FROM operating_system_vulnerabilities
+			WHERE cve = ?
+		) AS cve_table ON cm.cve = cve_table.cve
+		LEFT JOIN operating_system_vulnerabilities osv ON osv.cve = cm.cve
+		LEFT JOIN software_cve sc ON sc.cve = cm.cve
+		LEFT JOIN vulnerability_host_counts vhc ON cm.cve = vhc.cve
+`
 
 	freeSelectStmt := `
```

Comment:
updating this statement because the `cve_meta` table is not populated on Fleet free